Makefile

```diff
@@ -65,6 +65,7 @@ test: export MODULES_NAMESPACE?=fybrik-blueprints
 test: export CONTROLLER_NAMESPACE?=fybrik-system
 test: pre-test
 	go test -v ./...
+	CSP_PATH=$(ABSTOOLBIN)/fzn-or-tools go test -v ./manager/controllers/app -count 1
```

Comment:
Add the solver installation target as a dependency to the test target. If I run `make test` without installing the solver, I want it to be installed automatically.

Reply:
Do we want to add separate deploy with and without csp?

Reply:
@simanadler This target refers to testing manager when it runs locally. 
As for the deployment, I think we'll deploy with csp but run twice: with or without its usage (passing the env var in the command line).

Reply:
Added `fzn-or-tools` as a dependency of `pre-test`